.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install numpy cchardet openpyxl pandas
+        pip install pytest coverage pytest-cov codecov pathlib
+    - name: Test with pytest
+      run: |
+        pytest --cov-fail-under=80 --cov-branch
```

Comment:
We should keep the test command as `pytest`. Any command-line arguments can go into  [pytest.ini](https://github.com/kinverarity1/lasio/blob/master/pytest.ini).

What do `--cov-fail-under=80` and `--cov-branch` do - are they needed?

Reply:
`--cov-fail-under=80`  will report coverage failure if the test coverage is below 80%.   My current view, is this would be an undesirable constrant on Lasio at this time but go to remember if Lasio continuous integration activity picks up significantly.

Reply:
`--cov-branch`  is interesting.  It checks coverage on branching code (if/else, try/catch...) 

Reference: https://coverage.readthedocs.io/en/latest/branch.html

However, after tinkering with it,   I found that it is useful for seeing which files have a lot of branching (and thus should be given test attention, but it didn't provide enough information to go right to the branch locations.  So I didn't find it useful for running on every push. Thus I recommend leaving it out of Continuous Integration runs but it is good to run periodically and make test coverage imrprovements based on the files with high numbers of branching logic.

Reply:
Thanks